apps/builder/app/builder/features/project-settings/project-settings.tsx

```diff
@@ -20,27 +34,80 @@ const ProjectSettingsView = ({
   isOpen: boolean;
   onOpenChange: (isOpen: boolean) => void;
 }) => {
+  const [selectedSetting, setSelectedSetting] = useState<
+    (typeof settingNames)[number]
+  >(settingNames[0]);
+
   return (
     <Dialog open={isOpen} onOpenChange={onOpenChange}>
       <DialogContent
         // Left Aside panels (e.g., Pages, Components) use zIndex: theme.zIndices[1].
         // For a dialog to appear above these panels, both overlay and content should also have zIndex: theme.zIndices[1].
         css={{
-          width: theme.spacing[34],
+          width: "max-content",
```

Comment:
IMO this can lead cause serious width issues with Dialog having that max-width is set to none

Reply:
Also it's hard to detect what child gives width for the whole element

Reply:
done